File: 338. Counting Bits/cs/solution.cs
public class Solution {
    public int[] CountBits(int n) {
        List<int> result = new List<int>();
        result.Add(0);

        for(int i = 1; i <= n; i++) {
            int istring = Convert.ToString(i, 2).Replace("0", "").Length;
            result.Add(istring);
        }
          
        return result.ToArray();
    }
}

File: 338. Counting Bits/js/solution.js
/**
 * @param {number} n
 * @return {number[]}
 */
var countBits = function (n) {
    let result = [0];

    for (let i = 1; i <= n; i++) {
        let str = i.toString(2).replace(/0/g, "");
        result.push(str.length);
    }

    return result;
};